app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,31 @@
+class AttendancesController < ApplicationController
+
+  def index
+    @attendance = Attendance.new
+  end
+
+  def show
+
+  end
+
+  def new
+
+  end
+  
```

Comment:
Empty methods could be in on lin
```suggestion
  def show; end
  
  def new; end
```

Reply:
Ok